static/js/jira-configuration.js

```diff
@@ -171,3 +171,42 @@ window.onclick = function(event) {
 	}
 };
 
+$(".jiraConfiguration__deleteGitHubApp").click(function(event) {
+	event.preventDefault();
+	const uuid = $(event.target).data("app-uuid");
+
+	AP.context.getToken(function(token) {
+		$.ajax({
+			type: "DELETE",
+			url: `/jira/connect/enterprise/app/:${uuid}`,
```

Comment:
Adding the `:` would change the value, isn't it?
```suggestion
			url: `/jira/connect/enterprise/app/${uuid}`,
```